### DISCUSSION TITLE
Not able to create a new database.

### DISCUSSION THREAD

**Discussion by Tongch875** [pawelsalawa/sqlitestudio#4703] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.
![image](https://user-images.githubusercontent.com/80622218/221431721-6ffc6744-d3ed-4414-932a-af258fcb0008.png)
Not able to create a new database.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.
Windows 10

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.
3.4.2, 3.4.3

**Comment by pawelsalawa** [comment]
Why don't you pick a file to create it?

**Reply by miku239** [reply]
What if I don't have such a file?I need to install the SQLite platform, and then create it in the command line? And clearly there was this feature in the old version, I don't understand why it was removed.
